_pages/clients/javascript.md

```diff
@@ -25,7 +25,7 @@ You can include the JavaScript file as a script tag:
 
 #### Call an Algorithm
 
-Next, login to [Algorithmia](https://algorithmia.com/) to get your [API key](https://algorithmia.com/user#credentials):
+Next, login at [Algorithmia]({{ site.baseurl }}/) to get your [API key]({{ site.baseurl }}/user#credentials):
```

Comment:
"login to" = 168M google hits; "login at" = 2.5M

Reply:
I guess I don't know english well enough, haha. thanks